client/src/app/shared/components/FilterToolbar/FilterToolbar.tsx

```diff
@@ -30,6 +31,7 @@ export interface IBasicFilterCategory<
   key: TFilterCategoryKey;
   title: string;
   type: FilterType; // If we want to support arbitrary filter types, this could be a React node that consumes context instead of an enum
+  filterGroup: string;
```

Comment:
I think we'll probably need this to be optional, so it can be omitted for filters that don't have groups.